cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -256,7 +268,9 @@ protected void shellHandleIntent(Intent intent) {
       Log.i(TAG, "Ignoring command line params: can only be set when creating the activity.");
     }
 
-    if (MemoryPressureListener.handleDebugIntent(this, intent.getAction())) return;
+    if (MemoryPressureListener.handleDebugIntent(this, intent.getAction())) {
```

Comment:
If this is intentionally empty handler ( and is expected to be forever ) i would add a comment that states this. Otherwise, if we are expecting to do more around this, i would add a TODO

Reply:
google-java-format check showed me an error, it did not let me commit without fixing it. how did you make the commit?

Reply:
the error is gone.

Reply:
I'm unsure - i did run `pre-commit run --from-ref=origin/main --to-ref=HEAD` before sending it.

But regardless - i also didn't add this code and i'm not sure who did or what it is needed for - can you add a comment that states something about it, given you had to change it ?

Separately, we should also run `pre-commit run --all` and submit all formatting changes - @briantting could do this.